lib/ddlog_std.dl

```diff
@@ -752,6 +752,14 @@ function keys(m: Map<'K, 'V>): Vec<'K> {
     map_keys(m)
 }
 
+function values(m: Map<'K, 'V>): Vec<'V> {
+    map_values(m)
+}
+
+function nth(m: Map<'K, 'V>, n: usize): Option<'V> {
```

Comment:
Why not nth_value?

Reply:
Yeah, `nth_value` is less ambiguous.  I'll do that.